platform/platform-app/src/main/resources/features.xml

```diff
@@ -130,10 +130,10 @@
         <bundle dependency='true'>mvn:joda-time/joda-time/${jodatime2-bundle.version}</bundle>
         <bundle dependency='true'>mvn:com.sun.mail/javax.mail/${javax-mail.version}</bundle>
         <bundle dependency='true'>mvn:commons-codec/commons-codec/${commons-codec.version}</bundle>
-        <bundle dependency='true'>mvn:org.apache.servicemix.specs/org.apache.servicemix.specs.stax-api-1.0/${cxf.servicemix.specs.version}</bundle>
-        <bundle dependency='true'>mvn:com.fasterxml.jackson.core/jackson-core/${cxf.jackson.version}</bundle>
-        <bundle dependency='true'>mvn:com.fasterxml.jackson.core/jackson-databind/${cxf.jackson.version}</bundle>
-        <bundle dependency='true'>mvn:com.fasterxml.jackson.core/jackson-annotations/${cxf.jackson.version}</bundle>
+        <bundle dependency='true'>mvn:org.apache.servicemix.specs/org.apache.servicemix.specs.stax-api-1.0/${servicemix-specs-version}</bundle>
+        <bundle dependency='true'>mvn:com.fasterxml.jackson.core/jackson-core/${jackson2-version}</bundle>
+        <bundle dependency='true'>mvn:com.fasterxml.jackson.core/jackson-databind/${jackson2-version}</bundle>
+        <bundle dependency='true'>mvn:com.fasterxml.jackson.core/jackson-annotations/${jackson2-version}</bundle>
         <bundle dependency='true'>mvn:org.apache.servicemix.bundles/org.apache.servicemix.bundles.aws-java-sdk/${aws-java-sdk-bundle.version}</bundle>
```

Comment:
❓ Looks like camel added
```xml
<bundle dependency='true'>mvn:com.fasterxml.jackson.dataformat/jackson-dataformat-cbor/${jackson2-version}</bundle>
```
. Why wasn't that added here? https://github.com/apache/camel/compare/camel-2.19.0...camel-2.19.4#diff-dc9df8361e440f26d0cd182370b98df7 platforms/karaf/features/src/main/resources/features.xml

Reply:
Good catch, thanks!